BUILD.bazel

```diff
@@ -67,17 +125,60 @@ java_binary(
 java_binary(
     name = "code_generator_request_file_to_gapic_main",
     main_class = "com.google.api.generator.debug.CodeGeneratorRequestFileToGapicMain",
-    runtime_deps = [
-        "//src/main/java/com/google/api/generator",
-        "//src/main/java/com/google/api/generator/debug",
-        "//src/main/java/com/google/api/generator/gapic",
-        "@com_google_googleapis//google/api:api_java_proto",
-        "@com_google_googleapis//google/longrunning:longrunning_java_proto",
-        "@com_google_guava_guava",
-        "@com_google_protobuf//:protobuf_java",
-    ],
+    runtime_deps = MAIN_DEPS_DEBUG_RUNTIME_ONLY,
 )
 
+genrule(
+    name = "basic_proto_descriptor",
+    srcs = ["src/test/resources/basic.proto"],
+    outs = ["basic_proto.descriptor"],
+    cmd = "$(location @com_google_protobuf//:protoc) " +
+          "--include_source_info --include_imports --descriptor_set_out=$(OUTS) $(SRCS)",
+    message = "Generating proto descriptor",
+    tools = ["@com_google_protobuf//:protoc"],
+)
+
+[java_test(
```

Comment:
Can you also please add a test_suite target combining all the tests under single suit?